src/semeio/forward_models/__init__.py

```diff
@@ -54,7 +56,11 @@ def validate_pre_realization_run(
         return fm_step_json
 
     def validate_pre_experiment(self, fm_step_json: ForwardModelStepJSON) -> None:
-        pass
+        errors = DesignKWScript.validate_configuration(fm_step_json.get("argList")[0])
+        if errors:
+            raise ForwardModelStepValidationError(
```

Comment:
Updated the issue as we need this to be a warning first before we commit to having an error.